### DISCUSSION TITLE
Fuse F1 is blown

### DISCUSSION THREAD

**Discussion by WFeneberg** [Lenti84/VdMot_Controller#94] [Q&A] [answered]
My colleague managed to blow fuse F1. Where can I get the 1A fuse?

What type of fuse is it fast, slow etc.

**Comment by Lenti84** [comment, marked as answer]
It's a JFC1206-1100FS.
Rating 1A
Blows in 5 seconds at 2.5 overcurrent
I²t: 0.1

But I think any similar fuse will do the job.
Like ESKA 430.017 (Reichelt BELFUSE 430017).

**Comment by macbombastic** [comment]
I also managed to blow the fuse, at the first power up - where to search for the reason?

**Comment by Wookbert** [comment]
@Lenti84 Stupid question: Are there PTC Resettable Fuses available which meet the specs/demands? Else I’d probably switch to glass fuse holder like [two of these](https://www.lcsc.com/product-detail/C2927393.html), which makes fuse replacement much easier.
